packages/@sanity/default-layout/src/navbar/branding/Branding.tsx

```diff
@@ -15,15 +40,15 @@ function Branding(props: Props) {
   const logo = props.logo || BrandLogo
 
   return (
-    <div className={styles.root}>
-      {logo && <div className={styles.brandLogoContainer}>{createElement(logo)}</div>}
+    <Root padding={2} sizing="border">
```

Comment:
The padding should be `padding={3}`. The `sizing="border"` is unnecessary, I think?

Reply:
Thanks. I added `sizing="border"` because the previous implementation had that. Perhaps safe to keep it?